Source/debug.cpp

```diff
@@ -73,7 +73,7 @@ enum class DebugGridTextItem : uint16_t {
 	AutomapView,
 	dungeon,
 	pdungeon,
-	dflags,
+	Protected,

```

Comment:
Very minor suggestion: Is protected the best word here? Maybe locked would sound better? Feel free to ignore this comment.

Reply:
I think there's gonna be a bit of work on the dungeon generation code now so plenty of chances to address this in the future :D

Reply:
Yeah I was not in love with the name either, but couldn't come up with anything better. I sort of hope I can stick to some more user relevant changes, but going over this code is very interesting :)